### PR TITLE
feat: remove the "Contributors" list and re-arrange the "Help" info

### PR DESCRIPTION
**Pull request by ghost**
1. Make "Help" Info in verticle in ul...li style.
2. remove the contributors' list, this will be discussed at (https://github.com/nodejs/nodejs.org/issues/4043).
3. Ref from "README.md" in "CONTRIBUTING.md", because for newbies, they should know how to step-by-step building the nodejs.org.
4. Considering the random contributor is related to node/node.dev (https://nodejs.dev/, at the footer) instead of node.org itself, so remove it here.

Ref: https://github.com/nodejs/nodejs.org/issues/4000

---
捕获

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -58,3 +58,7 @@ resolution mechanism.
 Individuals with non-trivial contributions may be added as Collaborators.
 Collaborators are expected to follow this policy and continue to send pull
 requests and go through proper review.
+
+## How to contribute
+
+Please read more at [How to contribute](/README.md)
```
Comment (Trott) [inline CONTRIBUTING.md]:
```suggestion
Please read more at [How to contribute](/README.md#contributing)
```

**Review by Trott — APPROVED** [review]
This makes the help info easier to see/find. 👍

**Comment by XhmikosR** [comment]
This is broken on mobile...

**Comment by Trott** [comment]
> This is broken on mobile...

@XhmikosR It looks like the footer can be wider than the viewport and (in a sufficiently narrow viewport such as on a mobile device) there's no way to scroll right to see the links on the right hand side of the footer.

Is that what you're talking about? Is there something more as well that I'm missing?

**Comment by Trott** [comment]
> > This is broken on mobile...
> 
> @XhmikosR It looks like the footer can be wider than the viewport and (in a sufficiently narrow viewport such as on a mobile device) there's no way to scroll right to see the links on the right hand side of the footer.
> 
> Is that what you're talking about? Is there something more as well that I'm missing?

Created new issue for this at https://github.com/nodejs/nodejs.org/issues/4060.

**Pull request merged**